catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -205,59 +206,53 @@ public SolrQuery not(SolrQuery operand) {
 
     @Override
     public SolrQuery propertyIsFuzzy(String propertyName, String searchPhrase) {
-        String mappedPropertyName = getMappedPropertyName(propertyName,
-                AttributeFormat.STRING,
-                false);
-
-        StringBuilder phraseBuilder = new StringBuilder();
+        // On fuzzy searches, no text analysis is performed on the search phrase. Expect fuzzy
+        // terms to be case insensitive.
+        StringBuilder fuzzyPhrase = new StringBuilder();
+        fuzzyPhrase.append("(");
```

Comment:
I would consider using a `StringJoiner` here:
```
StringBuilder fuzzyPhrase = Stream.of(StringUtils.split(input))
                .map(s -> s.toLowerCase() + "~ ")
                .collect(Collectors.joining("+", "(+", ")"))
```
_N.B. the space after the tilde._